packages/xstate-wallet/src/constants.ts

```diff
@@ -37,3 +37,12 @@ export function tokenAddress(assetHolderAddress: string): string | undefined {
   // TODO: store mapping, implement lookup
   return assetHolderAddress;
 }
+
+function envVarToBoolean(envVar: string | undefined): boolean {
+  if (!envVar) return false;
+  const upperEnvVar = envVar.toLocaleUpperCase();
+  if (upperEnvVar === 'FALSE') return false;
+  if (upperEnvVar === 'TRUE') return true;
+  throw new Error(`Environment variable ${envVar} must be a boolean`);
+}
```

Comment:
Just wanted to point out that we have this utility in devtools:

https://github.com/statechannels/monorepo/blob/20112d7a213340b42429247fc195b885873fc599/packages/devtools/src/config/env.ts#L4-L21

Reply:
Good catch

Reply:
f170ebb7c